src/commands/cmd_bulk_insert.c

```diff
@@ -74,9 +74,9 @@ static void _MGraph_BulkInsert(void *args) {
 	argc -= 2; // skip "GRAPH.BULK [GRAPHNAME]"
 
 	if(_MGraph_Bulk_Begin(ctx, &argv, &argc,
-				rs_graph_name, graphname) != BULK_OK) goto cleanup;
+						  rs_graph_name, graphname) != BULK_OK) goto cleanup;
 
-	gc = GraphContext_Retrieve(ctx, rs_graph_name, false, false);
+	gc = GraphContext_Retrieve(ctx, rs_graph_name, false, true);
```

Comment:
We should create the key in `_MGraph_Bulk_Begin`

Reply:
If the key is empty, we will not instantiate a `GraphContext` without this argument.